### PR TITLE
Add dropdown to navbar

### LINKED ISSUE #118: Reorganize navigation bar with dropdown menus
# Summary

Reorganize site navigation bar to fit more options via dropdown menus

# Background

In FOSSRIT/fossrit.github.io#85, @ct-martin added a calendar page. It would be nice to add this to the navigation bar to make it more visible. However, the navigation bar is at capacity for what it can display. To mitigate, we should add dropdown sub-menus to existing options to accommodate more space and tie some things together.

# Details

Here is the breadcrumb structure I would like to see for the navigation bar:

* Home
* About
  * LibreCorps
  * Events
  * Projects
* Announcements
* Calendar
* Get Involved

# Outcome

Make new pages on the website more visible

### PR DESCRIPTION
**Pull request by ct-martin**
Fixes #118 

image
image

### CODE REVIEW

File: _data/tabs.yml [hunk 1]
```diff
@@ -2,19 +2,21 @@
   link: /
 
 - name: About
-  link: /about/
+  dropdown:
```
Comment (ct-martin) [inline _data/tabs.yml:5]: BTW, I intentionally didn't put extra newlines here b/c when I added them it was hard to see that they were part of the dropdown visually; this makes it clearly look like they're related

**Comment by ct-martin** [comment]
Re Travis: linking to `#` is the standard practice for doing this; you shouldn't be checking any URL that matches `^#.*`

**Comment by ct-martin** [comment]
Ok, so to make clear, I added the flag `allow_hash_href` to HTMLProofer. This allows the `href="#"` to pass checks. This won't break the intended purpose b/c hash links are only for reference to a location within the same page.

File: _data/tabs.yml (hunk 1 above)
Comment (jwflory) [reply _data/tabs.yml:5]: Makes sense to me. :+1:

**Pull request merged**